### ISSUE TITLE
Gang Sign still triggers after Amani Senai bounce

### ISSUE THREAD

**Issue by BobAloVskI** [mtgred/netrunner#4487]
Corp scored an agenda. Amani Senai triggered. Trace was successful. Corp bounced Gang Sign. Gang Sign then triggered.

I don't think Gang Sign should still trigger as it is no longer installed.

**Comment by NoahTheDuke** [comment]
Thanks for the report!

**Comment by NoahTheDuke** [comment]
I can't reproduce this. Do you have any additional information to help out?

**Comment by BobAloVskI** [comment]
![Test1 - Screenshot](https://user-images.githubusercontent.com/55002486/65762339-237ec300-e164-11e9-93e6-362cbf52284e.PNG)
![Test2 - Screenshot](https://user-images.githubusercontent.com/55002486/65762340-237ec300-e164-11e9-8341-8429e217aa7f.PNG)
[Test1 - Log.txt](https://github.com/mtgred/netrunner/files/3662255/Test1.-.Log.txt)
[Test2 - Log.txt](https://github.com/mtgred/netrunner/files/3662256/Test2.-.Log.txt)

Hello,

I was able to reproduce it but it only occurred when Leela was the runner's ID.

Test 1 was with Leela as runner and NEH as corp. Test 2 was with Gabe as runner (same deck except for ID) and NEH as corp. I have attached the log file and screenshot of the game for each test.

If you require further information let me know and I should be able to get it for you.

Regards,
Robert
